src/wp-admin/customize.php

```diff
@@ -84,18 +84,20 @@
 	}
 }
 
-$url       = ! empty( $_REQUEST['url'] ) ? sanitize_text_field( $_REQUEST['url'] ) : '';
-$return    = ! empty( $_REQUEST['return'] ) ? sanitize_text_field( $_REQUEST['return'] ) : '';
-$autofocus = ! empty( $_REQUEST['autofocus'] ) ? sanitize_text_field( $_REQUEST['autofocus'] ) : '';
+$url    = ! empty( $_REQUEST['url'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['url'] ) ) : '';
+$return = ! empty( $_REQUEST['return'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['return'] ) ) : '';
+$autofocus = ! empty( $_REQUEST['autofocus'] ) && is_array( $_REQUEST['autofocus'] )
+				? array_map( 'sanitize_text_field', wp_unslash( $_REQUEST['autofocus'] ) )
+				: array(); // Since `$_REQUEST['autofocus']` is an array.
```

Comment:
nit for readability:

```suggestion
$autofocus = ! empty( $_REQUEST['autofocus'] ) && is_array( $_REQUEST['autofocus'] )
	? array_map( 'sanitize_text_field', wp_unslash( $_REQUEST['autofocus'] ) )
	: array(); // Since `$_REQUEST['autofocus']` is an array.
```